### PR TITLE
add TL skeleton to build pipeline

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh [hunk 1]
```diff
@@ -34,17 +34,34 @@ fi
...
 
-az resource create --id $disk_resource_id  --is-full-object --location $LOCATION --properties "{\"location\": \"$LOCATION\", \
-  \"properties\": { \
-    \"osType\": \"$OS_TYPE\", \
-    \"creationData\": { \
-      \"createOption\": \"FromImage\", \
-      \"galleryImageReference\": { \
-        \"id\": \"${sig_resource_id}\" \
+if [[ ${ENABLE_TRUSTED_LAUNCH} == "True" ]]; then
+  az resource create --id $disk_resource_id  --is-full-object --location $LOCATION --properties "{\"location\": \"$LOCATION\", \
```
Comment (alexeldeib) [inline vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh:38]: hmm, we should figure out if we can extract this more cleanly, e.g. mutate a base template

Comment (alexeldeib) [reply vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh:38]: not for now

**Review by alexeldeib — APPROVED** [review]
some opportunities for later refactoring but lgtm

File: vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh (hunk 1 above)
Comment (anujmaheshwari1) [reply vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh:38]: will merge and iterate on this in follow up PR

**Comment by anujmaheshwari1** [comment]
successful test run: https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=61916011&view=results

**Pull request merged**